app/controllers/plans_controller.rb

```diff
@@ -38,9 +38,13 @@ def new
   def create
     @plan = Plan.new
     authorize @plan
-
+    
     @plan.principal_investigator = current_user.surname.blank? ? nil : "#{current_user.firstname} #{current_user.surname}"
-    @plan.data_contact = current_user.email
+    @plan.principal_investigator_email = current_user.email
+    
+    orcid = current_user.identifier_for(IdentifierScheme.find_by(name: 'orcid'))
```

Comment:
@xsrust I think the identifier_schemes table is convenient because we can tack on descriptions, urls and other values